src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/prisonperson/PrisonPersonDpsApiService.kt

```diff
@@ -21,13 +21,14 @@ class PrisonPersonDpsApiService(@Qualifier("prisonPersonApiWebClient") private v
     weightKilograms: Int?,
     appliesFrom: LocalDateTime,
     appliesTo: LocalDateTime?,
+    latestBooking: Boolean,
     createdAt: LocalDateTime,
     createdBy: String,
   ): PhysicalAttributesSyncResponse =
     webClient
       .put()
       .uri("/sync/prisoners/{prisonerNumber}/physical-attributes", prisonerNumber)
-      .bodyValue(syncPhysicalAttributesRequest(heightCentimetres, weightKilograms, appliesFrom, appliesTo, createdAt, createdBy))
+      .bodyValue(syncPhysicalAttributesRequest(heightCentimetres, weightKilograms, appliesFrom, appliesTo, latestBooking, createdAt, createdBy))
```

Comment:
When syncing physical attributes to DPS we now tell them which is the latest booking in case there are no active bookings.